src/annotator/anchoring/text-position.js

```diff
@@ -28,19 +28,21 @@ export function toRange(root, start, end) {
     root,
     NodeFilter.SHOW_TEXT,
     null, // filter
+
+    // @ts-ignore - Ignored parameter required for IE 11
     false // expandEntityReferences
   );
 
   let startContainer;
-  let startOffset;
+  let startOffset = 0;
   let endContainer;
-  let endOffset;
+  let endOffset = 0;
```

Comment:
This makes it clearer to TS that `startOffset` and `endOffset` will be numbers. They should always get set to numbers before the `while` loop below exits, but TS can't figure that out.